ykrt/src/compile/jitc_yk/codegen/x64/rev_analyse.rs

```diff
@@ -129,7 +128,7 @@ mod test {
         let alives = rev_analyse(&m).unwrap().0;
         assert_eq!(
             alives,
-            vec![3, 0, 5, 0, 0, 0]
+            vec![2, 0, 5, 0, 0, 0]
```

Comment:
What do these numbers mean and why did it change from 3 to 2?

Reply:
This tells us "what is the last instruction that uses the value produced by this instruction?" Because of ptradd inlining, the 3rd instruction is dead code, so the last instruction that uses this value is actually the 2nd instruction.

Reply:
Aha. Thank you.